### DISCUSSION TITLE
Story Automation

### DISCUSSION THREAD

**Discussion by KenQuin** [community/community#25209] [Actions]
<p>Is there a way to automate the creation of a story or issue? What I mean by this is, that when something bad happens during the execution of a program, report it through a story on github, withouth me doing it and share it with my colleagues and if anyone knows how to fix it, accept the story and document the problems.</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/kenquin">@kenquin</a> ,</p>
<p>Maybe you can try using the GitHub REST API <a href="https://developer.github.com/v3/issues/#create-an-issue" rel="nofollow noopener"><strong>Create an issue</strong></a> to create the issues in the workflow run.</p>
<p>For example, when you run the workflows to build, test and deploy your program, if one of the job has problem, set to run a job to execute the API to create an issue.</p>

**Comment by KenQuin** [comment]
<p>That is awsome!! Thanks so much!!!</p>
